editor/doc/editor_help.cpp

```diff
@@ -4368,6 +4368,22 @@ void EditorHelpBit::parse_symbol(const String &p_symbol, const String &p_prologu
 			symbol_hint = SYMBOL_HINT_ASSIGNABLE;
 		}
 		help_data = _get_property_help_data(class_name, item_name);
+
+		// Add copy note to built-in properties returning `Packed*Array`.
+		const DocData::ClassDoc *cd = EditorHelp::get_doc(class_name);
+		if (cd && !cd->is_script_doc) {
+			for (const DocData::PropertyDoc &prop : cd->properties) {
+				if (prop.name == item_name) {
+					if (packed_array_types.has(prop.type)) {
+						if (!help_data.description.is_empty()) {
+							help_data.description += "\n";
+						}
+						help_data.description += vformat(TTR("[b]Note:[/b] The returned array is [i]copied[/i] and any changes to it will not update the original property value. See [%s] for more details."), prop.type);
```

Comment:
This string should probably be placed in one place so it doesn't get out of sync with the one on line 2328, might even be good to come up with some way to unify it with the doc generation py files as well

I'd at least add a comment to this and the other line to make sure they are in sync